backend/app.ts

```diff
@@ -6,7 +6,7 @@ import session from "express-session";
 import bodyParser from "body-parser";
 import cors from "cors";
 import paginate from "express-paginate";
-import { graphqlHTTP } from "express-graphql";
+import { createHandler } from "graphql-http/lib/use/express";
```

Comment:
I'd like to see this import renamed to something more descriptive, e.g.

```suggestion
import { createHandler as graphqlHandler } from "graphql-http/lib/use/express";
```

or similar. This helps with the readability of the `use` block below